services/api/service.go

```diff
@@ -41,9 +41,11 @@ import (
 )
 
 const (
-	ErrBlockAlreadyKnown  = "simulation failed: block already known"
-	ErrBlockRequiresReorg = "simulation failed: block requires a reorg"
-	ErrMissingTrieNode    = "missing trie node"
+	RelayActualGasLimit     = 30_000_000
+	RelayFictitiousGasLimit = 1_000_000_000
```

Comment:
The same effect could be realized with a lower limit since the only requirement is that 30m is less than half. A safer limit might be 90m or 120m. 90m is half of Polygon's gas-denominated capacity.

Reply:
this is a good point. I wanted to leave some room to allow complex flexing behavior for `RelayActualGasLimit` (trivial example, doubling or tripling if the proposer reward is an outlier). but i guess we could keep the fictitious limit around 100mm and change it in tandem if we want to implement flexing later

Reply:
made the change in [51dffae](https://github.com/flashbots/mev-boost-relay/pull/519/commits/51dffae142876f5f82cb2202fb7d1c55ea6f10de)